SKX/metrics/perf/skylakex_metrics_perf.json

```diff
@@ -274,1294 +274,1476 @@
     },
     {
         "BriefDescription": "This category represents fraction of slots where the processor's Frontend undersupplies its Backend. Frontend denotes the first part of the processor core responsible to fetch operations that are executed later on by the Backend part. Within the Frontend; a branch predictor predicts the next address to fetch; cache-lines are fetched from the memory subsystem; parsed into instructions; and lastly decoded into micro-operations (uops). Ideally the Frontend can issue Pipeline_Width uops every cycle to the Backend. Frontend Bound denotes unutilized issue-slots when there is no Backend stall; i.e. bubbles where Frontend delivered no uops while Backend could have accepted them. For example; stalls due to instruction-cache misses would be categorized under Frontend Bound.",
-        "MetricExpr": "( IDQ_UOPS_NOT_DELIVERED.CORE / ( ( 4 ) * ( ( CPU_CLK_UNHALTED.THREAD_ANY / 2 ) if #SMT_on else ( CPU_CLK_UNHALTED.THREAD ) ) ) )",
-        "MetricGroup": "TmaL1;PGO;TopdownL1;tma_L1_group",
+        "MetricExpr": "IDQ_UOPS_NOT_DELIVERED.CORE / slots",
+        "MetricGroup": "BvFB;BvIO;TmaL1;PGO;TopdownL1;tma_L1_group;Slots",
         "MetricName": "tma_frontend_bound",
-        "ScaleUnit": "100%"
+        "ScaleUnit": "100%",
+        "Threshold": "tma_frontend_bound > 15"
```

Comment:
Here is the existing perf version:
```
    {
        "BriefDescription": "This category represents fraction of slots where the processor's Fronte
nd undersupplies its Backend",
        "MetricExpr": "IDQ_UOPS_NOT_DELIVERED.CORE / tma_info_thread_slots",
        "MetricGroup": "PGO;TmaL1;TopdownL1;tma_L1_group",
        "MetricName": "tma_frontend_bound",
        "MetricThreshold": "tma_frontend_bound > 0.15",
        "MetricgroupNoGroup": "TopdownL1",
        "PublicDescription": "This category represents fraction of slots where the processor's Frontend undersupplies its Backend. Frontend denotes the first part of the processor core responsible to fetch operations that are executed later on by the Backend part. Within the Frontend; a branch predictor predicts the next address to fetch; cache-lines are fetched from the memory subsystem; parsed into instructions; and lastly decoded into micro-operations (uops). Ideally the Frontend can issue Pipeline_Width uops every cycle to the Backend. Frontend Bound denotes unutilized issue-slots when there is no Backend stall; i.e. bubbles where Frontend delivered no uops while Backend could have accepted them. For example; stalls due to instruction-cache misses would be categorized under Frontend Bound. Sample with: FRONTEND_RETIRED.LATENCY_GE_4_PS",
        "ScaleUnit": "100%"
    },
```
* It looks like Threshold should be MetricThreshold.
* `slots` rather than `tma_info_slots`
* In the description  `Sample with: FRONTEND_RETIRED.LATENCY_GE_4_PS` isn't present.

Reply:
I've renamed the "Threshold" field to "MetricThreshold"